### ISSUE TITLE
GLPI 9.2

### ISSUE THREAD

**Issue by vincemd22** [ddurieux/glpi_physicalinv#5]
Bonjour, avez vous prévu une compatibilite avec GLPI 9.2 ?
Cordialement

Hello, have you planned a compatibility with GLPI 9.2 ?
Best Regards

**Comment by ddurieux** [comment]
oui, je regarde ça rapidement, mais ça devrait fonctionner sans trop de mal

**Comment by Bena555** [comment]
Bonjour,
Nous attendons également patiemment la mise à jour de votre plug-in :)
Cordialement.

**Comment by DGermany** [comment]
Hello, 

we're waiting too :) Do you have an update-Date?! 

Best Regards
